.github/workflows/pytest.yml

```diff
@@ -0,0 +1,40 @@
+name: Run the unit tests on PR or commit
+
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
+
+jobs:
+  ci:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-22.04, macos-latest]
+        # TODO: pixeltable currently fails on python 3.12.
+        python-version: ["3.9", "3.10", "3.11"]
```

Comment:
Can we include 3.8? I think that's still the most frequently used version.

Could be in a follow-on, but then let's leave a todo.

Reply:
Sure, I'll add 3.8.